static/sass/_pattern_cve.scss

```diff
@@ -46,13 +72,15 @@
   }
 
   .cve-table .icon-container__text {
-    overflow: hidden;
-    text-overflow: ellipsis;
+    // overflow: hidden;
+    // text-overflow: ellipsis;
     width: 100%;
+    word-wrap: break-word;
   }
```

Comment:
Would it be okay to remove `overflow: hidden; and text-overflow: ellipsis` and add `word-wrap: break-word;` instead? Because the texts are not fully shown on medium size screens

Reply:
I think this is good. Better to see the wrapped contents then hiding it.